starter_code/hangman.js

```diff
@@ -1,36 +1,51 @@
 var hangman;
 
-// function Hangman() {
 
-// }
-
-// Hangman.prototype.getWord = function () {
-
-// };
-
-// Hangman.prototype.checkIfLetter = function (keyCode) {
-
-// };
-
-// Hangman.prototype.checkClickedLetters = function (key) {
+function Hangman() {
+  this.words = [];
+  this.secretWord = '';
+  this.letters = [];
+  this.guessedLetter = '';
+  this.errorsLeft = 10;
+}
+
+Hangman.prototype.getWord = function () {
+  this.secretWord = toString(this.words[Math.floor(Math.random*this.words.length)]);
+  return this.secretWord;
+};
 
-// };
+Hangman.prototype.checkIfLetter = function (keyCode) {
+  if (keyCode <= 43) {
+    return false;
+  } else {
+    return true;
+  }
+};
 
-// Hangman.prototype.addCorrectLetter = function (i) {
+Hangman.prototype.checkClickedLetters = function (key) {
+  if (this.letters.includes(key) === true) {
+    return false;
+  } else {
+    return true;
+  }
```

Comment:
You don't need the `else` statement since when you have a `return` the function will end in that moment. So if the `if` is true, the `return` will end the function.